internal/scripts/scripts.go

```diff
@@ -171,7 +177,11 @@ func (c *ContentValue) Write(thread *starlark.Thread, fn *starlark.Builtin, args
 
 	// No mode parameter for now as slices are supposed to list files
 	// explicitly instead.
-	err = os.WriteFile(fpath, fdata, 0644)
+	err = c.Create(&fsutil.CreateOptions{
+		Path: fpath,
+		Data: bytes.NewReader(fdata),
+		Mode: fs.FileMode(0644),
```

Comment:
This changes the mode in the final fs, right? Can we add a test or change an existing one to show that creating a text file (or whatever) with 0645 and then mutating it results in a final mode of 0644?

Reply:
It would not change the permission of an existing file. So `0644` would only be set for files that do not exist already in the system. I will add a test to make sure that the original perm is retained if the file exists previously.

Reply:
Added a test.